docs/platforms/javascript/common/session-replay/index.mdx

```diff
@@ -48,6 +48,49 @@ To set up the integration, add the following to your Sentry initialization. Ther
 
 <PlatformContent includePath="session-replay/setup" />
 
+<PlatformSection supported={["javascript"]} notSupported={["javascript.deno", "javascript.bun", "javascript.wasm", "javascript.capacitor", "javascript.electron"]}>
```

Comment:
Do we need to add the `supported` part here? 🤔  or would it be enough to add the excludes?

Reply:
Ah yeah removed, this means that only javascript browser is supported (and not frameworks).